### PR TITLE
Added basic scripts to provide conversions to other formats

### PR DESCRIPTION
**Pull request by ghost**
I've just thrown together some scripts to make it easy to port markdown source to other formats.
So far LaTeX, HTML and Man are supported, but it should be trivial to add more

### CODE REVIEW

File: other-formats/convert.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    echo    " ${directories[@]}"
+    echo -e "===========================\n"
+
+    for i in ${directories[@]}; do
+	cat "chapters.txt" |
+	    while read -r current_file; do
+		echo "Working on directory '$i', file '$current_file'"
+
+		# Make sure the 'gen.sh' exists
+		if [ ! -f "$i/gen.sh" ]; then
```
Comment (feroldi) [inline other-formats/convert.sh]: Use POSIX compliant features. I'd like to have these things run on any shell.

File: other-formats/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ - HTML
+ - Man page
+
+## Running the Script
+
+Very simple:
+
+    $ ./convert.sh
+
+This will convert the markdown files from book into all of the supported formats.
```
Comment (feroldi) [inline other-formats/README.md:17]: Can we have this convert only to what the user wants? I.e. add an option for which formats you'd like to generate.

File: other-formats/convert.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```
Comment (feroldi) [inline other-formats/convert.sh]: Use `#!/bin/sh` instead. This path is POSIX, and I need these scripts to be POSIX compliant anyway.

**Review by feroldi — CHANGES_REQUESTED** [review]
Make sure these points are met, and I'll take care of the rest. Thanks!

**Comment by ghost** [comment]
I'm pretty sure these commits should fix the POSIXness. I have also added a way to specify a certain format for generation.

Let me know if there's anything else I need to fix up

**Comment by feroldi** [comment]
after modifying these shell scripts for a while, I realized we're trying to imitate `makefiles`. I'm gonna work on it from now on, and then, after coming up with something, close this PR in order to introduce the new build environment.

Thanks for your help!

edit: Maybe I'll just stick with this branch.

**Comment by ghost** [comment]
This is a very valid point :D

I'll be closing this one then

**Pull request closed without merging**